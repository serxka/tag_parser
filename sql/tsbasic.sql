CREATE EXTENSION tag_parser;
SELECT alias,token,lexemes FROM ts_debug('tag_parser', 'red,red_hair,red-hair,red hair,red_hairs');

