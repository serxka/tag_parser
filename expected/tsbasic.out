CREATE EXTENSION tag_parser;
SELECT alias,token,lexemes FROM ts_debug('tag_parser', 'red,red_hair,red-hair,red hair,red_hairs');
 alias |   token   |   lexemes
-------+-----------+--------------
 tag   | red       | {red}
 break | ,         |
 tag   | red_hair  | {red_hair}
 break | ,         |
 tag   | red-hair  | {red-hair}
 break | ,         |
 tag   | red hair  | {"red hair"}
 break | ,         |
 tag   | red_hairs | {red_hairs}
(9 rows)

